packages/react/public/index.html

```diff
@@ -23,7 +23,7 @@
     -->
     <title>UI Template Gallery</title>
   </head>
-  <body class="dx-viewport">
+  <body class="dx-viewport theme-light">
```

Comment:
Let's move the theme className to the wrapper div in the App.tsx, we won't need to manipulate DOM in this case.